packages/convertkit-sdk/src/convertkit.ts

```diff
@@ -101,13 +111,17 @@ export async function tagSubscriber(email: string, tagId: string) {
     }),
   })
     .then((res) => res.json())
-    .then(({subscription}: any) => {
-      return subscription.subscriber
+    .then((jsonRes: any) => {
+      const result = TagSubscriberResponseSchema.safeParse(jsonRes)
+      if (!result.success) {
+        return undefined
+      }
+      return result.data.subscription.subscriber
```

Comment:
use Zod to validate the exact data we want from this function. the types flow where this gets used which made it easier to notice a logic inconsistency.